commands/queue.js

```diff
@@ -37,11 +37,11 @@ module.exports = class extends Command {
 			const embed = new RichEmbed()
 				.setTitle('Current Queue')
 				.setDescription(
-					oneLine`Singing in ${formatChannelName(
-						'#' + (queue.voiceChannel ? queue.voiceChannel.name : '<nowhere>')
+					oneLine`Singing in ${formatVoiceChannelName(
+						(queue.voiceChannel ? queue.voiceChannel.name : '<nowhere>')
 					)},
 					 speaking in ${formatChannelName(
```

Comment:
I think you forgot to change this function name. Can u do a grep to change all of them?